webapp/src/components/expanded_view/component.tsx

```diff
@@ -594,16 +679,33 @@ export default class ExpandedView extends React.PureComponent<Props, State> {
         const raiseHandText = isHandRaised ? 'Lower hand' : 'Raise hand';
         const participantsText = 'Show participants list';
 
+        let chatToolTipText = this.props.isRhsOpen && this.props.rhsSelectedThreadID === this.props.threadID ? 'Click to close chat' : 'Click to open chat';
+        const chatToolTipSubtext = '';
+        const chatDisabled = Boolean(this.props.channel?.team_id) && this.props.channel.team_id !== this.props.currentTeamID;
```

Comment:
![CleanShot 2022-11-03 at 02 03 16@2x](https://user-images.githubusercontent.com/11724372/199664501-7b08ef11-d1a1-42ba-a8f5-1b6f462ceed5.png)


Thank you for catching this! Indeed this shouldn't happen. Very simple fix, added a call channel team_id check, should now open expanded window in the correct team even if you switch to another team.

The constraint is because of the team scoping that is present in RHSThread. Technically it could be removed, but it's a safeguard against a lot of other potential issues where the currentTeamId is different from the current selected thread team id.

Reply:
Awesome, works great now, thanks!